### ISSUE TITLE
Make sure opimizations are aware about wrtbar/rdbars

### ISSUE THREAD

**Issue by cathyzhyi** [eclipse/omr#3033]
Wrtbar and rdbar opcodes are added in https://github.com/eclipse/openj9/pull/3138 and more investigation is needed to make sure the opts working for a normal store/load also work for wrtbar/rdbar.

**Comment by 0xdaryl** [comment]
@cathyzhyi:  just to clarify, this is for performance rather than functional correctness?

**Comment by cathyzhyi** [comment]
@0xdaryl yes, it's for performance and the opcodes are functionally working now.

**Comment by cathyzhyi** [comment]
the PR disabled PRE and FieldPrivatizer for functional correctness https://github.com/eclipse/omr/pull/3224
